### PR TITLE
Website frontpage

### PR DESCRIPTION
**Pull request by ewels**
Ported the website from https://github.com/ewels/MegaQC_website into the sphinx docs homepage.

Tried to keep the files added to a minimum, hopefully not too much boilerplate code here. Still needed some assets though.

Tested locally and looks fine 🌟

### CODE REVIEW

**Comment by ewels** [comment]
While I was here I also removed the readme from the sphinx docs. See https://github.com/ewels/MegaQC/issues/217

File: docs/source/_templates/index.html [hunk 1]
```diff
@@ -1,1730 +1,215 @@
...
+							<div id="logo">
+								<h1><a href="index.html"><img src="_static/images/MegaQC_logo_darkbg.svg" title="MegaQC"></a></h1>
+								<p>Collect and visualise data from multiple MultiQC reports</p>
+							</div>
+
+						<!-- Nav -->
+							<nav id="nav">
+								<ul>
+									<li class="current"><a href="index.html">Home</a></li>
+									<li><a href="docs/index.html" title="Read the MegaQC documentation">Docs</a></li>
```
Comment (Zethson) [inline docs/source/_templates/index.html:27]: I was always using `{{ pathto("docs/index") }}` or whatever, but hardcoding it should be fine as well.

File: docs/source/_templates/index.html [hunk 2]
```diff
@@ -1,1730 +1,215 @@
...
+					<div class="title">Try before you buy</div>
+					<div id="footer" class="container">
+						<header class="style1">
+							<h2>Still not sure?</h2>
+							<p>
+								Investing time into a new tool is a bit of a commitment, so to be sure <br class="mobile-hide" />
+                that MegaQC does what you want you can check out some further resources:
+							</p>
+              <ul class="actions">
+                <li><a href="#" class="button style2">Read the docs</a></li>
```
Comment (Zethson) [inline docs/source/_templates/index.html:187]: This button should lead to 
```suggestion
                <li><a href="docs/index.html" class="button style2">Read the docs</a></li>
```

**Review by Zethson — APPROVED** [review]
One minor fix.

LGTM, although it should be filled with more mature content soon. Looks more placeholdery now than before :)

**Pull request merged**